hook.php

```diff
@@ -869,6 +869,9 @@ function plugin_pre_item_purge_glpiinventory($parm)
     $rule = new RuleMatchedLog();
     $rule->deleteByCriteria(['itemtype' => $itemtype, 'items_id' => $items_id]);
 
+    $lf = new LockedField();
+    $lf->deleteByCriteria(['itemtype' => $itemtype, 'items_id' => $items_id]);
```

Comment:
I do not get why it is done in plugin. If an item is purged, why GLPI is not cleaning locks by itself ?

Reply:
To avoid GLPI minor version dependencie

Reply:
Ok, so this PR is useless now we can close it=)